mule-artifact-it/mule-deployer-it/src/test/java/integration/test/mojo/CloudHubDeploymentTest.java

```diff
@@ -175,9 +182,9 @@ public String getApplicationStatus() {
 
     @Override
     public Boolean run() {
-      Application application = cloudHubClient.getApplication(applicationName);
-      applicationStatus = application.status;
-      if (application != null && expectedStatus.equals(application.status)) {
+      Application application = cloudHubClient.getApplications(applicationName);
+      applicationStatus = application.getStatus();
+      if (application != null && expectedStatus.equals(application.getStatus())) {
```

Comment:
Could you please check whether application is null before invoking getStatus? Otherwise this will throw an NPE

Reply:
done